UI/src/view/navigationpanel/speedLimitBox.js

```diff
@@ -0,0 +1,109 @@
+(function(root) {
+  root.SpeedLimitBox = function (selectedSpeedLimit) {
+    ActionPanelBox.call(this);
```

Comment:
This one also can extend from the LinearAssetBox

Reply:
After analysing it, it's not needed to extend from LinearAssetBox, since we have to redefine everything